### ISSUE TITLE
DDPM

### ISSUE THREAD

**Issue by liurob2000** [Shark-NLP/DiffuSeq#79]
Thank  @summmeer  for sharing the code.  Does DiffuSeq use DDPM model as the foundation?

**Comment by summmeer** [comment]
Yes. For detailed information, please refer to the original paper.

**Comment by BIT-MJY** [comment]
Hi @summmeer ,

Thanks for your great work. Have you tried to infer only with one or two steps rather than the total steps?

**Comment by summmeer** [comment]
@BIT-MJY Hi, YES, using v2 to speed-up, one or two steps also works. For a simple task, we can almost maintain the performance, while for more complex tasks there might be performance drop. You can refer to our follow-up work DoT Figure 3 (https://arxiv.org/pdf/2402.07754)
